### PR TITLE
Assignment on Spiral order for the 2D arrays

### PR DESCRIPTION
**Pull request by rajeshvlega9**
Spiral order output for the test case.
image

### CODE REVIEW

File: Week-6/spiral.js [hunk 1]
```diff
@@ -0,0 +1,104 @@
...
+    for (let i = fcol; i <= lcol; i++) numbers.push(matrix[frow][i]);
+    frow++;
+    // last column
+    for (i = frow; i <= lrow; i++) numbers.push(matrix[i][lcol]);
+    lcol--;
+    //bottow row
+    for (i = lcol; i >= fcol; i--) numbers.push(matrix[lrow][i]);
+    lrow--;
+    //first col
+    for (i = lrow; i >= frow; i--) numbers.push(matrix[i][fcol]);
```
Comment (RajeshSivanesan) [inline Week-6/spiral.js:77]: @rajeshvlega9 just see whether decrementing or incrementing values not exceeding the max value. add a boundary condition to avoid errors

Comment (rajeshvlega9) [reply Week-6/spiral.js:77]: Hi Rajesh, the boundary condition is checked in the while loop, 
image

please find the test results for the below cases
image

**Review by RajeshSivanesan — APPROVED** [review]
